feedback/styles/styles.scss

```diff
@@ -0,0 +1,683 @@
+// display nav mobile change//
+$display-nav: 750px;
+
+// JS action classes (*****where to place action classes in relation to other css content****)//
+[data-visible="false"] {
+    display: none;
+}
+
+/*google fonts*/
+$sans-serif: 'Noto Sans TC',
+sans-serif;
+
+/*google fonts, weights*/
+$bold-weight: 700;
+$medium-weight: 400;
+$regular-weight: 300;
+
+/*caps text*/
+$capsspacing: 2px;
+$capssize: 13px;
+
+/*footer text*/
+$footersize: 12px;
+$footersec-toppadding: 3vw;
+
+/*far left page margin on desktop layout*/
+$left-margin: 4vw;
+
+//colors//
+$white: #F9F4ED;
+$blue:#1b2d34;
+$pink:#C1423F;
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+a:hover {
```

Comment:
This can be nested inside the anchor above, like so:
```
a {
    text-decoration: none;
    color: inherit;
    &:hover {
        text-decoration: underline;
    }
}
```